malaria24/ona/models.py

```diff
@@ -654,9 +655,20 @@ class ReportedCase(models.Model):
     ehps = models.ManyToManyField('Actor', blank=True)
     form = models.ForeignKey('OnaForm', null=True, blank=True)
 
+    def normalize_msisdn(self, mobile_number):
+        valid_prefix = '[+27]'
+        if re.match('^' + valid_prefix + '*([0-9]{9})$', mobile_number):
+            return mobile_number
+        else:
+            return '+27' + mobile_number[1:]
```

Comment:
Is there a test for this line? (A case where the msisdn doesn't start with +27)

Reply:
I was told that MalariaConnect only has SA reported cases. Do we have other country codes?

Reply:
Yes I meant if it starts with `0`, but as mentioned in slack I checked the coverage and it's fine. 🙂

Reply:
Thanks for pointing it out